src/common.rs

```diff
@@ -16,8 +17,17 @@ impl Warning {
             line,
             check_name,
             message,
+            is_fixed: false,
         }
     }
+
+    pub fn line_number(&self) -> usize {
+        self.line.number
+    }
+
+    pub fn mark_as_fixed(&mut self, val: bool) {
```

Comment:
I apologize, it seemed to me that we are only setting the meaning of `true`, so such a renaming would make sense.
That is, it seems to me that this setter should not take a value, but simply set true independently.

If need set false, `set_fixed` really more suitable. Or add second method `mark_as_unfixed` :)

Reply:
There is no `Option` anymore, so it seems that we can remove the argument (because the default value is `false`).

Reply:
Done.